handler/commentHandler.go

```diff
@@ -332,6 +370,8 @@ func parse(body, commandTrigger string) *types.CommentAction {
 		commandTrigger + "unlock":             unlockConstant,
 		commandTrigger + "set milestone: ":    setMilestoneConstant,
 		commandTrigger + "remove milestone: ": removeMilestoneConstant,
+		commandTrigger + "review: ":           assignReviewerConstant,
```

Comment:
This was my suggestion, but on second thoughts how about `reviewer` and also `set reviewer` as an alias?

Reply:
`set reviewer` sounds good